Source/Main.as

```diff
@@ -1,4 +1,8 @@
 bool g_visible = true;
+float g_dt = 0;
+float g_prev_speed = 0;
+array<float> g_acc = {0, 0, 0, 0};
+int g_idx = 0;
```

Comment:
I don't like these being globals. Why are they not part of `DashboardAcceleration`?

Reply:
Huh, I had an issue before where state wasn't being properly saved in `DashboardAcceleration` so I kept things global, but now that I'm rechecking I might have done something wrong when I was trying to use them as `DashboardAcceleration` locals. I'll fix this.